doc/rst/usingchapel/chplenv.rst

```diff
@@ -146,6 +146,7 @@ CHPL_HOST_ARCH
         ========  =============================================================
         x86_64    64-bit AMD and Intel processors
         aarch64   64-bit ARM processors
+        arm64     64-bit ARM processors on Mac laptops
```

Comment:
There are desktops as well, so pulling from https://support.apple.com/en-us/HT211814 I might say "64-bit ARM-based Apple silicon" (for some reason, they use "silicon" instead of "processor". Not sure if that's just marketing or because it's a SoC that combines a bunch of things. Or maybe "64-bit ARM processors on Mac computers" if we want something like the current wording.

Reply:
My understanding is that `arm64` is a synonym for `aarch64` that happens to currently be used by Apple but that is not exclusively used by Apple.

* [Debian releases](https://wiki.debian.org/DebianBullseye) call it `arm64`
* [this S.O post explains some of the history](https://stackoverflow.com/questions/31851611/differences-between-arm64-and-aarch64) including that the Linux support for 64-bit ARM was called arm64.
* it is my understanding that `aarch64` is a more official name (from ARM) and `arm64` is a more informal name.

As a result, I would lean towards just having the docs describe `arm64` as "alternative name for aarch64" or something like that.